pkg/config/config.go

```diff
@@ -17,11 +17,11 @@ import (
 )
 
 type ProviderConfig struct {
-	KubeConfigPath    string                                 `koanf:"kubeConfigPath"`
-	ContainerRegistry string                                 `koanf:"containerRegistry"`
-	RunnerNamespace   string                                 `koanf:"runnerNamespace"`
-	PodTemplate       corev1.PodTemplateSpec                 `koanf:"podTemplate"`
-	Flavors           map[string]corev1.ResourceRequirements `koanf:"flavors"`
+	KubeConfigPath string `koanf:"kubeConfigPath"`
+	//ContainerRegistry string                                 `koanf:"containerRegistry"`
```

Comment:
let's drop this comment as well - we have git history ;-)

Reply:
damn, has probably crept in during development 🤦‍♂️
resolved with [35c0e15](https://github.com/mercedes-benz/garm-provider-k8s/pull/42/commits/35c0e15085aedf54567e50b0967029ee887196df)